test/topology_custom/test_view_build_status.py

```diff
@@ -175,3 +175,101 @@ async def test_view_build_status_migration_to_v2(request, manager: ManagerClient
 
     result = await cql.run_async("SELECT * FROM system.view_build_status_v2")
     assert len(result) == 6
+
+# Test that when removing a node from the cluster, we clean its rows from
+# the view build status table.
+@pytest.mark.asyncio
+async def test_view_build_status_cleanup_on_remove_node(manager: ManagerClient):
+    node_count = 4
+    servers = await manager.servers_add(node_count)
+    cql = manager.get_cql()
+    await wait_for_cql_and_get_hosts(cql, servers, time.time() + 60)
+    await manager.servers_see_each_other(servers)
+
+    await create_keyspace(cql)
+    await create_table(cql)
+    await create_mv(cql, "vt1")
+    await create_mv(cql, "vt2")
+
+    await wait_for_view(cql, "vt1", node_count)
+    await wait_for_view(cql, "vt2", node_count)
+
+    result = await cql.run_async("SELECT * FROM system.view_build_status_v2")
+    assert len(result) == node_count * 2
```

Comment:
Considering that `system_distributed.view_build_status` is now a virtual table reading from `system.view_build_status_v2`, these lines check the same thing twice. However, I'm not seeing any explicit test which confirms that reading from these 2 tables gives the same result, so this segment could serve as a test for it.

Though it would probably be best if we just made a separate test for this and read from just one table here (and in some other tests)

Reply:
I added a virtual table test